resources/views/home.blade.php

```diff
@@ -66,6 +66,11 @@ class="client-icon h-auto w-44 m-auto grayscale hover:grayscale-0 transform tran
                          class="client-icon h-auto w-44 m-auto grayscale hover:grayscale-0 transform transition hover:scale-110"
                          alt="KGHM"/>
                 </div>
+                <div class="m-6">
```

Comment:
I would prefer to move these to some resource/config/whatever. It will be easier to update/reorder/manage these.

Reply:
moved them